app/models/ht_institution.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# We're not really interested in editing or viewing this table,
+# but we want to allow HTUser access to a human-readable version of
+# the institution.
+class HTInstitution < ApplicationRecord
+  self.primary_key = 'sdrinst'
```

Comment:
`sdrinst` is deprecated; use `inst_id` instead.

Reply:
Should be addressed in second commit. Tests pass locally.